packages/node_modules/@webex/plugin-meetings/src/meetings/util.js

```diff
@@ -72,4 +75,87 @@ MeetingsUtil.checkForCorrelationId = (deviceUrl, locus) => {
   return false;
 };
 
+
+/**
+ * Will check to see if specified media codec is supported.
+ * @async
+ * @returns {Boolean}
+ */
+MeetingsUtil.hasH264Codec = async () => {
+  let hasCodec = false;
+
+  try {
+    const pc = new window.RTCPeerConnection();
+    const offer = await pc.createOffer({offerToReceiveVideo: true});
+
+    if (offer.sdp.match(/^a=rtpmap:\d+\s+H264\/\d+/m)) {
+      hasCodec = true;
+    }
+    LoggerProxy.logger.info(`Meetings:util#hasH264Codec --> H.264 codec ${hasCodec ? 'is' : 'is not'} supported.`);
+  }
+  catch (error) {
+    LoggerProxy.logger.error('Meetings:util#hasH264Codec --> Error creating peerConnection for H.264 test.');
+  }
+
+  return hasCodec;
+};
+
+
+/**
+ * Notifies the user whether or not the codec
+ * is present. Will continuously check until
+ * max duration.
+ * @async
+ * @param {Number} firstChecked Timestamp in milliseconds.
+ * @returns {undefined}
+ */
+MeetingsUtil.checkMediaPlayback = async (firstChecked) => {
```

Comment:
This function name is a bit ambiguous to me. Let's just call it what it is:
```suggestion
MeetingsUtil.checkH264Support = async (firstChecked) => {
```

Reply:
Sounds good to me.

Reply:
Albeit, there is a function with the same name inside PeerConnectionManager.